README.md

```diff
@@ -226,3 +227,5 @@ guidelines][contributing].
 [refaster]: https://errorprone.info/docs/refaster
 [refaster-rules-bigdecimal]: https://github.com/PicnicSupermarket/error-prone-support/blob/master/error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/BigDecimalRules.java
 [refaster-rules]: https://github.com/PicnicSupermarket/error-prone-support/blob/master/error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/
+[reproducible-builds-badge]: https://img.shields.io/badge/Reproducible_Builds-ok-success?labelColor=1e5b96
```

Comment:
How was the color selected? :smile:

(I guess that eventually it'd be cool to have dynamic badges that go red if the latest release is no longer reproducible :sparkles:.)

Reply:
color was chosen after Reproducible Builds logo